### DISCUSSION TITLE
Github Pages

### DISCUSSION THREAD

**Discussion by rodrigoricky** [retypeapp/retype#184] [General]
The site doesn't seem to work like the one in the localhost, the CSS currently do not work: live example, https://thecoom.github.io/thecoom/
![image](https://user-images.githubusercontent.com/89636767/138717768-1858bd16-9182-4782-852d-4af2e852c66e.png)

**Comment by geoffreymcgill** [comment]
Hi @rodrigoricky. You likely need to set the [`url`](https://retype.com/configuration/project/#url) config with `url: thecoom.github.io/thecoom/` in your projects `retype.yml` file. That should fix it.

```yml
url: thecoom.github.io/thecoom/
```

Here are some additional docs on the `url` config with GitHub Pages. See https://retype.com/guides/github-actions/#set-a-url. 

Hope this helps.
